mods/ra/rules/vehicles.yaml

```diff
@@ -284,6 +265,9 @@ HARV:
 		Crushes: wall, mine, crate, infantry
 	RevealsShroud:
 		Range: 4c0
+	-WithFacingSpriteBody:
+	-AutoSelectionSize:
+	-RenderSprites:
```

Comment:
Because of the line below: harvesters trade `RenderSprites` for (the soon to be deprecated (see #8553)) `RenderHarvester`.
@reaperrr might be a good idea to think about which of the PRs you want to go in first, so that there isn't any junk YAML left afterwards.